client/src/Star.jsx

```diff
@@ -1,14 +1,68 @@
-import React, { useState } from 'react';
-import styled from 'styled-components';
+import React from 'react';
+import PropTypes from 'prop-types';
+// import styled from 'styled-components';
 
-export default function Star() {
+export default function Star({ score }) {
```

Comment:
You mentioned this on Slack already, but we'll need some way to get this `score` in order to plug this into other components, but it's non-breaking and everything else looks good so I feel good merging this!

Reply:
Thanks! I will make a ticket so we don't forget about this.